lunes_cms/cms/models/document.py

```diff
@@ -63,15 +63,13 @@ class Document(models.Model):
     feedback = GenericRelation(Feedback)
 
     @property
-    def converted(self, content_type="audio/mpeg"):
+    def converted(self):
```

Comment:
The parameter is simply removed becuase `converted` is being used only once and without argument in the whole project.